src/index.html

```diff
@@ -6,8 +6,36 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="page" data-qa="card">
+      <div class="product-card">
+        <div class="product-card-image"></div>
+        <div class="product-card-text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="product-card-code">Product code: 195434</div>
+        <div class="product-card-rating">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="stars-text">Reviews: 5</div>
+        </div>
+        <div class="product-card-menu">
+          <div class="menu-text">Price:</div>
+          <div class="menu-count">$2,199</div>
+        </div>
+        <a
+          class="product-card-button"
+           href="#"
+           data-qa="hover"
+        >BUY</a>
+      </div>
```

Comment:
let's make some formatting:
```suggestion
      <div class="product-card">
        <div class="product-card-image"></div>

        <div class="product-card-text">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </div>

        <div class="product-card-code">Product code: 195434</div>

        <div class="product-card-rating">
          <div class="stars">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>

          <div class="stars-text">Reviews: 5</div>
        </div>

        <div class="product-card-menu">
          <div class="menu-text">Price:</div>
          <div class="menu-count">$2,199</div>
        </div>

        <a
          class="product-card-button"
           href="#"
           data-qa="hover"
        >
          BUY
        </a>
      </div>
```